src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -375,6 +375,49 @@ def get_validate_platform(cmd, os_type, platform):
     return platform_os, platform_arch, platform_variant
 
 
+def get_custom_registry_credentials(cmd, auth_mode=None, login_server=None, username=None, password=None):
+    """Get the credential object from the input
+    :param str auth_mode: The login mode for the source registry
+    :param str login_server: The login server of custom registry
+    :param str username: The username for custom registry
+    :param str password: The password for custom registry
+    """
+    source_registry_credentials = None
+    if auth_mode is not None:
+        SourceRegistryCredentials = cmd.get_models('SourceRegistryCredentials')
+        source_registry_credentials = SourceRegistryCredentials(login_mode=auth_mode)
+
+    custom_registries = {}
+    if login_server is not None:
```

Comment:
Do you want to check for null or empty?

Reply:
`null`. Cases where `login_server` is None is where the user only set the `auth_mode` to a value. Cases where we deal with credentials (`login_server` is never empty because it is then a required property).

Reply:
--login-server ""